EPR.Antivirus/EPR.Antivirus.Data/Enums/FileType.cs

```diff
@@ -6,5 +6,6 @@ public enum FileType
     CompanyDetails = 2,
     Brands = 3,
     Partnerships = 4,
-    Subsidiaries = 5
+    Subsidiaries = 5,
+    CompaniesHouse = 6
```

Comment:
Why are we adding numbers to an enum?
They automatically increment

Reply:
Numbered enums are extension of existing code. To be possibly looked at changing in future as tech debt.